shared/common-adapters/hot-key.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+import {bind, unbind} from '../util/mousetrap'
+// hook for hotkeys
+
+export function useHotKey(keys: Array<string> | string, cb: (key: string) => void) {
+  React.useEffect(() => {
+    bind(
+      keys,
+      (e: React.BaseSyntheticEvent, key: string) => {
+        e.stopPropagation()
+        key && cb(key)
+      },
+      'keydown'
+    )
+    return () => {
+      unbind(keys)
+    }
+    // eslint-disable-next-line
+  }, []) // on mount only
```

Comment:
Don't want a component to be able to change its hotkeys by changing what keys it passes to this hook?

Reply:
this is more or less a port of the previous version. we never dynamically mutate the keys so i'm keeping it simple for now